app/views/form.html

```diff
@@ -46,14 +46,11 @@
     </keyword-input>
   </div>
 
-  <h3>Droits attendus</h3>
+  <h3>Valeurs attendues</h3>
 
-  <p>
-    Attention : veillez à supprimer ici les droits que vous ne testez pas.
-    <br>Veillez aussi à ajouter les droits dont vous testez que le montant calculé doit être nul (en mettant "0" comme valeur attendue).
```

Comment:
All values are there by default, and I don't remember cases where a user would not have tested a null amount. The issue seems to be more often that they don't remove values they don't intend to test.

Unless we have evidence of user need, I'd remove as much text as possible to simplify the process.
You have interacted over Ludwig more often than I have. What is your impression?